examples/managed-catalog/create-table1.json

```diff
@@ -0,0 +1,32 @@
+{
+	"catalog_name": "default",
+	"schema_name": "default",
+	"name": "my_table",
+	"comment": "something really insightful",
+	"table_type": "MANAGED",
+	"data_source_format": "DELTA",
+	"columns": [
+		{
+			"name": "cust_id",
+			"data_type": "INT",
+			"nullable": false,
+			"position": 0
+		},
+		{
+			"name": "name",
+			"data_type": "STRING",
+			"comment": "Short name",
+			"position": 1
+		},
+		{
+			"name": "some_flag",
+			"data_type": "BOOLEAN",
+			"nullable": false,
+			"position": 2
+		}
+	],
+	"properties": {
+		"first_key": "first value",
+		"second_key": "second value"
+	}
+}
```

Comment:
nit, we might want to update those example jsons someday, since they are not working right now with the updated column info